src/components/PopoverMenuMultiSelect/index.tsx

```diff
@@ -0,0 +1,66 @@
+import React, { useState } from 'react';
+import { ListItemText } from '@mui/material';
+import { DropdownItem } from '../types';
+import Icon from '../Icon/Icon';
+import './styles.scss';
+import Popover from '../PopoverMenu/Popover';
+
+export type PopoverMenuMultiSelectProps = {
+  anchorElement: HTMLElement | null;
+  initialSelection: any[];
+  menuAlign?: 'left' | 'center' | 'right';
+  onChange: (selection: any[]) => void;
+  options: any[];
```

Comment:
use DropdownItem prop and only send values back? Same for initial selection. The labels are not required for selection.

Reply:
I changed options to `Section[]` to match the `Popover` api, but I left `initialSelection` as an `any[]`, since that matches the type of the `onChange` callback argument.

Reply:
That seems like the right thing to do 👍 
Changes look great, thanks for the iterations!! 🎉